controller/src/main/java/io/pravega/controller/fault/ControllerClusterListener.java

```diff
@@ -79,9 +97,12 @@ protected void startUp() throws Exception {
                     case HOST_REMOVED:
                         log.info("Received controller cluster event: {} for host: {}", type, host);
                         taskSweeper.sweepOrphanedTasks(host.getHostId());
-                        if (eventProcessorsOpt.isPresent() && eventProcessorsOpt.get().isRunning()) {
```

Comment:
This is because we have now moved the `wait until ready/running` check before registering the cluster listener. On  becoming ready, we first attach the cluster listener and then sweep the orphaned readers/tasks/transactions. Hence, whenever a cluster event is received, eventProcessors are already running, and `isRunning` check is redundant.